src/request-handler/lib/collapse_include.js

```diff
@@ -1,4 +1,5 @@
-export default function (includes) {
+export default function (input) {
+  const includes = Array.isArray(input) ? input : input.split(',');
```

Comment:
[It might be worth talking to @cowboy about this :P](http://benalman.com/code/projects/jquery-bbq/examples/deparam/)